pkg/deployer/transformer/target.go

```diff
@@ -32,5 +30,9 @@ func (g *targetTransformerGenerator) GetName() string {
 
 // TransformConfig is a nop
 func (t *targetTransformer) TransformConfig(ctx context.Context) error {
-	return t.hc.SetHalConfigProp("deploymentEnvironment.location", t.svc.GetNamespace())
+	err := t.svc.GetSpinnakerConfig().SetHalConfigProp("deploymentEnvironment.location", t.svc.GetNamespace())
+	if err != nil {
+		return err
+	}
+	return t.svc.GetSpinnakerConfig().SetHalConfigProp("deploymentEnvironment.type", "Operator")
```

Comment:
Are we supporting now this new deployment type? Are there any side effects or other places (like Halyard) that need to be updated?

Reply:
We already had the `Operator` deployment type. We now use it explicitly in Halyard. It's basically `Distributed` but without an account check  (you can deploy without having an account referenced)